Earthfile

```diff
@@ -1,20 +1,25 @@
+VERSION 0.6
+
 all:
     BUILD +all-test
     BUILD +all-integration-test
     BUILD +npm
 
 all-test:
-    BUILD --build-arg ELIXIR=1.9.4 --build-arg OTP=21.3.8.24 +test
-    BUILD --build-arg ELIXIR=1.12.1 --build-arg OTP=24.0.2 --build-arg RUN_INSTALLER_TESTS=1 +test
+    BUILD --build-arg ELIXIR=1.10.0 --build-arg OTP=21.3.8.24 +test
+    BUILD --build-arg ELIXIR=1.13.4 --build-arg OTP=24.3.4 --build-arg ALPINE=3.15.3 --build-arg RUN_INSTALLER_TESTS=1 +test
 
 test:
     FROM +test-setup
+    COPY --dir installer ./
```

Comment:
Fix `+test *failed* | ** (Mix) Cannot compile dependency :phx_new because it isn't available, please ensure the dependency is at "installer"`